### PR TITLE
Fixes #255

### LINKED ISSUE #255: Min/max scale doesn't match Scratch 2.0
Currently it's set to 5%/500% respectively.

However, it appears to have been set based on a function of costume width/height:

https://github.com/LLK/scratch-flash/blob/a931df6ab30e34a1cb2d6f7288f99df1c8f7f49b/src/scratch/ScratchSprite.as#L253

To pull that off, I think we'll need to provide costume/width height from the renderer.

### PR DESCRIPTION
**Pull request by CosmicWebServices**
I based it off of https://github.com/LLK/scratch-flash/blob/a931df6ab30e34a1cb2d6f7288f99df1c8f7f49b/src/scratch/ScratchSprite.as#L253 so that the min/max is like it was in scratch 2

### CODE REVIEW

**Comment by thisandagain** [comment]
Thanks! Looks like you likely have a syntax error somewhere which is causing the build to fail. Try doing a clean `rm -rf node_modules && npm install` followed by running the test suite via `npm test` prior to the next commit.

**Comment by CosmicWebServices** [comment]
OK I am sadly on my chromebook atm but I will do it later...

**Comment by CosmicWebServices** [comment]
Ah hah! I forgot an ending parenthesis!

**Comment by CosmicWebServices** [comment]
nope... still not the problem :(

**Comment by CosmicWebServices** [comment]
OK, I know the issue now... I will do it later today...

File: src/sprites/rendered-target.js [hunk 1]
```diff
@@ -250,7 +250,9 @@ RenderedTarget.prototype.setSize = function (size) {
         return;
     }
     // Keep size between 5% and 535%.
-    this.size = MathUtil.clamp(size, 5, 535);
+    var sizeMin = Math.min(1, Math.max(5 / costume.width, 5 / costume.height));
+    var sizeMax = Math.min((1.5 * 480) / costume.width, (1.5 * 360) / costume.height);
```
Comment (dekrain) [inline src/sprites/rendered-target.js]: Didn't you mean `this.costume` ?

Comment (CosmicWebServices) [reply src/sprites/rendered-target.js]: Yes

Comment (CosmicWebServices) [reply src/sprites/rendered-target.js]: Fixed

**Comment by dekrain** [comment]
Now good :smile:

**Comment by CosmicWebServices** [comment]
Yay! Thanks so much @dekrain!
@thisandagain I am ready to be checked again.

**Comment by dekrain** [comment]
@TheBrokenRail ?? Where?

**Comment by CosmicWebServices** [comment]
@dekrain Whoops sorry! I put the wrong mention!

**Comment by tmickel** [comment]
@CosmicWebServices I'm skeptical that this could work, since `this.costume` seems undefined.

**Comment by tmickel** [comment]
Yes - verified that this breaks the resize functionality. "Uncaught TypeError: Cannot read property 'width' of undefined"

**Comment by CosmicWebServices** [comment]
Yeah I will fix this... later

**Pull request closed without merging**